portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,40 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+<meta charset="utf-8">
+<title>My Portfolio</title>
+<link rel="stylesheet" href="style.css">
+<div id="header-content">
+  <div id="image-container">
+    <img src="images/main-selfie.jpg" alt="Picture of me" id="header-selfie">
+    <button id="cycle-button" onclick="cyclePictures()"> Cycle Pictures </button>
+  </div>	
+  <div id="greeting-container">
+    <h1>Welcome to my portfolio</h1>
+    <h3>My name is Malachi Brewer. I am a sophomore at the University of Georgia.</h3>
+    <h3>Welcome Buds! I created this site to house random pieces of information about me for anyone who might be curious. 
```

Comment:
It would be nice to break these lines a little earlier. Try to keep the horizontal width under ~120 characters, bonus points for keeping it under ~80 characters. This makes it easier to view in the GitHub UI and compare diffs in GitHub and editors.

Reply:
I changed all of my paragraphs so they are easier to read in the code. Let me know if it makes it look a little awkward with how I split some sentences up in awkward spots.